feat: support only build specified environment (#3059)
web-infra-dev/rsbuild: website/docs/en/guide/basic/cli.mdx

  --port <port>         specify a port number for Rsbuild Server to listen
  --host <host>         specify the host that the Rsbuild Server listens to
  -c --config <config>  specify the configuration file, can be a relative or absolute path

>   --environment <name>  specify the environment during multi-environment

Reviewer: ```suggestion
  --environment <name>  specify the name of environment to build
```
